app/assets/stylesheets/custom.css.scss

```diff
@@ -0,0 +1,16 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+/* universal */
+.navbar-default {
+    background-color: #8bc9ee;
+    border-color: #e7e7e7;
+}
+
+.navbar-default .navbar-nav > li > a {
+    color: #2734D5;
```

Comment:
Color literals like `#2734D5` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#2734D5` should be written as `#2734d5`<br>Line should be indented 2 spaces, but was indented 4 spaces